kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx

```diff
@@ -66,6 +67,25 @@ describe('Overview', () => {
     mockClearTopicMessages.mockClear();
   });
 
+  it('at least one replica was rendered', () => {
+    setupComponent({
+      ...defaultProps,
+      underReplicatedPartitions: 0,
+      inSyncReplicas: 1,
+      replicas: 1,
+    });
+    expect(screen.getByLabelText('replica-info')).toBeInTheDocument();
+  });
+
+  it('renders replica cell with props', () => {
+    render(<ReplicaCell leader />);
+    expect(screen.getByLabelText('replica-info')).toBeInTheDocument();
```

Comment:
i think element repeats twice we can put it in a variable

Reply:
fixed